source/services/helper.js

```diff
@@ -320,7 +320,9 @@
           elementIsTreeNode: function (element) {
             return typeof element.attr('ui-tree-node') !== 'undefined';
           },
-
+          elementIsEmptyTree: function (element) {
+            return element.attr('class') == "angular-ui-tree-empty";
```

Comment:
Could this respect empty placeholder class from tree configuration and be written as follows?
```javascript
  return element.attr('class') == treeConfig.emptyTreeClass;
```